containerize/Program.cs

```diff
@@ -200,7 +200,24 @@
     string _rid = context.ParseResult.GetValueForOption(ridOpt) ?? "";
     string _ridGraphPath = context.ParseResult.GetValueForOption(ridGraphPathOpt) ?? "";
     string _localContainerDaemon = context.ParseResult.GetValueForOption(localContainerDaemonOpt) ?? "";
-    await ContainerBuilder.Containerize(_publishDir, _workingDir, _baseReg, _baseName, _baseTag, _entrypoint, _entrypointArgs, _name, _tags, _outputReg, _labels, _ports, _envVars, _rid, _ridGraphPath, _localContainerDaemon).ConfigureAwait(false);
+    await ContainerBuilder.ContainerizeAsync(
+        _publishDir,
+        _workingDir,
+        _baseReg,
+        _baseName,
+        _baseTag,
+        _entrypoint,
+        _entrypointArgs,
+        _name,
+        _tags,
+        _outputReg,
+        _labels,
+        _ports,
+        _envVars,
+        _rid,
+        _ridGraphPath,
+        _localContainerDaemon,
+        context.GetCancellationToken()).ConfigureAwait(false);
```

Comment:
I think we need https://github.com/dotnet/sdk-container-builds/issues/172 to get cancellation working. The cancellation middleware might not be available by default.
I need to dive into `ToolTask` implementation to see how it is cancelled.

Reply:
`ToolTask` kills the process, so cancellation unfortunately is not happening.